index.html

```diff
@@ -76,9 +76,9 @@
             <li><a href="#signup">Sign Up</a></li>
             <li><a href="#tickets">Tickets</a></li>
             <li><a href="#cfp">Call For Proposals</a></li>
+            <li><a href="#geektrain">Geek Train</a></li>
```

Comment:
I gotta get back at this and am going to redo a few bits.

I'll actually bump the Geek Train to a separate landing page, that just links off of the schedule, etc.